packages/paste-core/components/button/src/index.tsx

```diff
@@ -153,6 +153,7 @@ const ButtonContents: React.FC<React.PropsWithChildren<ButtonContentsProps>> = (
         justifyContent={buttonVariantHasBoundingBox ? null : 'center'}
         columnGap="space20"
         alignItems="center"
+        width="100%"
```

Comment:
question: I wonder if this will have any unintended downstream conseuqneces?

Reply:
I don't believe it will, but Storybook/Website VRT should catch anything if it does.